### PR TITLE
Download right cargo executable based on platform

### PR DESCRIPTION
**Pull request by gleyba**
fix: download right cargo executable for the current platform
fix: Cargo.toml search path

### CODE REVIEW

File: blackjack.bzl [hunk 1]
```diff
@@ -8,6 +8,10 @@ def blackjack(name=None, manifest="Cargo.toml"):
       name = name,
       srcs = ["@blackjack//:src/bin/blackjack.rs"],
       aliases = {"@blackjack//:blackjack_lib": "blackjack"},
+      args = [
```
Comment (wildarch) [inline blackjack.bzl:11]: This is really neat, I had no idea this attribute existed! Is it a common attr for all binary rules?

**Review by wildarch — APPROVED** [review]
Thanks Gleb!

**Comment by wildarch** [comment]
If we merge this, it would be great if we also do CI runs for the new platforms. Without that I suspect it will be hard to maintain  support for all three platforms.

**Comment by SirWindfield** [comment]
@gleyba @wildarch Seeing that https://github.com/wildarch/blackjack/pull/9 is based on this PR and supersedes it, I think that this can be closed :)

**Comment by SirWindfield** [comment]
Merged via https://github.com/wildarch/blackjack/pull/9. Thanks @gleyba for the great work :)

**Pull request closed without merging**